src/app/layout/components/header/header.component.ts

```diff
@@ -1479,7 +1479,7 @@ export class HeaderComponent implements OnInit, OnDestroy, AfterViewInit {
   }
   checkMenuTokenInfo() {
     if (isPlatformBrowser(this.platformId))
-      window.open(environment.domainName + '/wallet/token-info?crypto=SATT', '_self');
+      window.open(environment.domainName + '/wallet/token-info', '_self');
   }
```

Comment:
Function Naming: The name checkMenuTokenInfo() implies that this function is checking something, but the main action it performs is opening a new window or tab. You may want to consider renaming this function to better reflect its main action. For instance, openTokenInfoInNewTab() could be a more descriptive name.